app/assets/javascripts/TortoiseJS/agent/editor.coffee

```diff
@@ -15,7 +15,10 @@ window.EditorWidget = Ractive.extend({
     """
     <div class="netlogo-code-container netlogo-widget">
       {{# !readOnly }}
-        <button class="netlogo-widget" on-click="recompile">compile</button>
+        <div class="netlogo-editor-button-container netlogo-widget" id="editor-buttons">
+          <button class="netlogo-tab editor-buttons netlogo-editor-button" id="editor-buttons" on-click="recompile">Compile</button>
+          <button class="netlogo-tab editor-buttons netlogo-editor-button" id="editor-buttons" on-click="exportnlogo">Export to NetLogo...</button>
+        </div>
```

Comment:
[Duplicate IDs are bad.](http://programmers.stackexchange.com/a/127180)  On top of that, it doesn't look to me like these IDs are ever actually used for anything, anyway...?